go/vt/vttablet/onlineddl/executor.go

```diff
@@ -1179,7 +1182,7 @@ func (e *Executor) validateAndEditAlterTableStatement(ctx context.Context, onlin
 					// in the same statement
 					extraAlterTable := &sqlparser.AlterTable{
 						Table:        alterTable.Table,
-						AlterOptions: []sqlparser.AlterOption{opt},
+						AlterOptions: []sqlparser.AlterOption{opt, sqlparser.AlgorithmValue("COPY")},
```

Comment:
Is it worth declaring the algorithm values as constants versus hard-coded strings?